rnd/docker-compose.yml

```diff
@@ -50,8 +50,11 @@ services:
       - app-network
 
   rest_server:
-    image: autogpt_server:latest
-    command: ["poetry", "run", "rest"]
+    build:
+      context: ../
+      dockerfile: rnd/autogpt_server/Dockerfile
+      target: server
+    command: ["sh", "-c", "sleep 5 && python -m autogpt_server.rest"]
```

Comment:
Why the change from `poetry run rest`?

Reply:
What's the 5-second delay for?

Reply:
5 second delay is to try and avoid intermittent prisma bugs.

change from poetry is to make sure its using the correct path - because the env vars are not carried over to this image

Reply:
I'm testing out the removal of sleep based on @aarushik93 comments